.chloggen/refactor-bridge-config.yaml

```diff
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: OpAMP Bridge
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: This PR simplifies the bridge's configuration and logging by renaming and removing fields.
+
+# One or more tracking issues related to the change
+issues: [1368]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  `components_allowed` => `componentsAllowed`
+  :x: `protocol` which is now inferred from endpoint
```

Comment:
what does `:x:` mean?  Does it mean that is has been removed?

Reply:
Yes